src/components/Navigation.js

```diff
@@ -46,19 +46,19 @@ function Navigation() {
         <div className="collapse navbar-collapse" id="navbarSupportedContent">
           <ul className="navbar-nav mr-auto">
             <li className="nav-item">
-              <NavLink to="/wallet/" exact className="nav-link" activeClassName="active" activeStyle={{ fontWeight: 'bold' }}>{t`Wallet`}</NavLink>
```

Comment:
The `NavLink` elements had to be adapted for the Router upgrade. Its style was moved to the main `scss` file.